content/influxdb/v2.3/write-data/no-code/load-data.md

```diff
@@ -77,3 +78,125 @@ Load CSV or line protocol data by uploading a file or pasting the data manually
+
+5. Under **Write Destination**, select an existing InfluxDB bucket to write data to or click **+ Create bucket**. For more information, see [Create a bucket](/influxdb/cloud/organizations/buckets/create-bucket/).
+
+#### Define parsing rules
+
+{{% note %}}
+JSON parsing is faster and more efficient than string parsing. We recommend using JSON parsing when your MQTT messages are in JSON format.
+{{% /note %}}
+
+- Under **Define Data Parsing Rules**, select one of the following MQTT data formats:
+
+   - **Line protocol** (no configuration required)
+   - **JSON**. To define parsing rules to ingest JSON data, click the **JSON** tab below.
+   - **String**. To define parsing rules to ingest String data, click the **String** tab below.
+
+{{< tabs-wrapper >}}
+{{% tabs %}}
+
+[Line protocol](#)
+[JSON](#)
+[String](#)
+
+{{% /tabs %}}
+
+<!-------------------------------- BEGIN Line protocol -------------------------------->
+{{% tab-content %}}
+Use line protocol to write data into InfluxDB. 
+
+Select a **Timepstamp precision** from the dropdown menu:
+   - **MS**: Milliseconds
+   - **S**: Seconds
+   - **US**: Microseconds
+   - **NS**: Nanoseconds
+
+{{% /tab-content %}}
+
+<!-------------------------------- BEGIN JSON -------------------------------->
+{{% tab-content %}}
+
+Associate **JSON** key/value pairs with **InfluxDB elements** (measurements, timestamps, fields, or tags).
+
+1. On the **Setting Up - MQTT Connector** page, under **Data Format**, do the following:
+
+  1. (Optional) In the **JSON path to timestamp** field, specify the path in the MQTT message to the JSON key that holds the timestamp. For example, `"time":1653998899010000000`. Otherwise, InfluxDB automatically assigns a timestamp when messages are ingested into InfluxDB.
+
+   {{% warn %}}
+   **Important**: Configure the timestamp format that matches the format in your messages.
+   {{% /warn %}}
+
+  2. Under **Measurement**, enter the **JSON path** (start with `$.`) to assign the InfluxDB measurement key, for example, `$.device_id` or `$.device_information.device_id` for a nested measurement key.
+  3. Select the **Data Type** for the measurement.
+  4. Specify the JSON paths to tag and field names as needed, and then select the data type for the tag or field. At least one field is required. Note, JSON paths with arrays are supported, for example, `$.device_information.errors_encountered[0].error_number`.
+
+{{% /tab-content %}}
+
+<!-------------------------------- BEGIN String -------------------------------->
+{{% tab-content %}}
+
+Associate **String** key/value pairs with **InfluxDB elements** (measurements, timestamps, fields, or tags).
+
+1. On the **Setting Up - MQTT Connector** page, under **Data Format**, do the following:
+
+  1. (Optional) In the **Regex pattern to find timestamp** field, enter the regex to find the timestamp in the MQTT message.  Otherwise, InfluxDB automatically assigns a timestamp when messages are ingested into InfluxDB.
```

Comment:
```suggestion
  1. (Optional) In the **Regex pattern to find timestamp** field, enter the regex (regular expression) to find the timestamp in the MQTT message.  Otherwise, InfluxDB automatically assigns a timestamp when messages are ingested into InfluxDB.
```